.travis.yml

```diff
@@ -9,7 +9,7 @@ go:
 
 services:
   - docker
-
+  
```

Comment:
@Lucy911, is the addition of this whitespace valuable? if not, may be best to keep it out of the file

small thing, but if you end up pushing another commit to the branch, may be worth changing